### PR TITLE
Prepare 2.12.6 release

### PR DESCRIPTION
**Pull request by isc-bsaviano**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## [2.12.6] 23-Jul-2024
+- Enhancements
+  - Support command stepping in debugger (#1385)
```
Comment (gjsjohnmurray) [inline CHANGELOG.md:5]: Maybe mention what version of server platform this feature requires.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
...
+  - Add `Compile` command to server-side file explorer (#1389)
+  - Associate unit test debug sessions with the test runs (#1395)
+  - Add command for opening InterSystems documents (#1398)
+  - Set file readonly if class is deployed or if server-side source control reports it not editable (off by default) (#1399)
```
Comment (gjsjohnmurray) [inline CHANGELOG.md:9]:
```suggestion
  - Always make isfs document readonly if class is deployed. Support `objectscript.serverSourceControl.respectEditableStatus` setting to make it readonly if server-side source control reports it not editable. (#1399)
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]
Please see inline comments.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,11 +2,12 @@
...
 ## [2.12.6] 23-Jul-2024
 - Enhancements
-  - Support command stepping in debugger (#1385)
+  - Support command stepping in debugger (requires InterSystems IRIS 2023.1.5, 2024.1.1+, or 2024.2+) (#1385)
   - Add `Compile` command to server-side file explorer (#1389)
   - Associate unit test debug sessions with the test runs (#1395)
   - Add command for opening InterSystems documents (#1398)
-  - Set file readonly if class is deployed or if server-side source control reports it not editable (off by default) (#1399)
+  - Make server-side class readonly if it's deployed (#1399)
+  - Add new `objectscript.serverSourceControl.respectEditableStatus` setting to make server-side file readonly if source control reports it i not editable (#1399)
```
Comment (gjsjohnmurray) [inline CHANGELOG.md:10]:
```suggestion
  - Add new `objectscript.serverSourceControl.respectEditableStatus` setting to make server-side file readonly if source control reports it is not editable (#1399)
```

**Review by gjsjohnmurray — CHANGES_REQUESTED** [review]

**Review by isc-rsingh — APPROVED** [review]

**Pull request merged**